features/facebookImport/test/ministory/report-metadata.test.js

```diff
@@ -92,18 +99,33 @@ describe("Report metadata analysis", () => {
     });
 
     it("has correct polyPod version in analysis", async () => {
-        expect(analysis._polyPodVersion).toBe(MOCKED_POD_RUNTIME_VERSION);
+        expect(reportData.polyPodVersion).toBe(MOCKED_POD_RUNTIME_VERSION);
     });
 
     it("has correct polyPod version in JSON report", async () => {
         expect(jsonReport.data.polyPodVersion).toBe(MOCKED_POD_RUNTIME_VERSION);
     });
 
     it("has correct polyPod runtime in analysis", async () => {
-        expect(analysis._polyPodRuntime).toBe(MOCKED_POD_RUNTIME);
+        expect(reportData.polyPodRuntime).toBe(MOCKED_POD_RUNTIME);
     });
 
     it("has correct polyPod runtime in JSON report", async () => {
         expect(jsonReport.data.polyPodRuntime).toBe(MOCKED_POD_RUNTIME);
     });
 });
+
+function getReportNameFromAnalaysis(analysis) {
+    return analysis.name.replace("Analysis", "Report");
+}
+
+function getReportData(analysisReport) {
+    const analysisData = analysisReport.reports[analysisKeys.reportMetadata];
+    return {
+        fileSize: analysisData.fileSize,
+        filesCount: analysisData.filesCount,
+        preferedLanguage: analysisData.preferedLanguage,
+        polyPodRuntime: analysisData.polyPodRuntime,
+        polyPodVersion: analysisData.polyPodVersion,
+    };
```

Comment:
Nit: I'd probably just do `return { ...analysisData }` - or if I get away with it just return `analysisData` by reference :D If the filtering is important, I'd leave it as-is though, filtering object by key looks more convoluted than this.